app/assets/stylesheets/govuk-component/_previous-and-next-navigation.scss

```diff
@@ -33,62 +35,47 @@
 
       .pagination-part-title {
         @include core-27($line-height: (33.75 / 27));
-        margin-bottom: 0.1em;
         display: block;
       }
-
-    }
-
-    &.next-page a:before,
-    &.previous-page a:before {
-      margin-top: -4px;
-      display: block;
-      width: 30px;
-      height: 38px;
-      content: " ";
-    }
-
-    &.next-page a:before {
-      background: transparent image-url("govuk-component/arrow-sprite.png") no-repeat -102px -11px;
-      margin-right: -32px;
-      float: right;
     }
+  }
 
-    &.previous-page a:before {
-      background: transparent image-url("govuk-component/arrow-sprite.png") no-repeat -20px -11px;
-      margin-left: -32px;
-      float: left;
-    }
+  .previous-page {
+    float: left;
+    text-align: left;
+  }
 
-    &.previous-page {
-      float: left;
-      text-align: left;
-    }
+  .next-page {
+    float: right;
+    text-align: right;
+  }
 
-    &.previous-page a {
-      padding: 0.75em 0 0.75em 3em;
+  @include media-down(mobile) {
+    .previous-page,
+    .next-page {
+      float: none;
+      width: 100%;
     }
 
-    &.next-page {
-      float: right;
+    .next-page a {
       text-align: right;
     }
+  }
 
-    &.next-page a {
-      padding: 0.75em 3em 0.75em 0;
-    }
+  .pagination-icon {
+    display: inline-block;
+    margin-bottom: 1px;
+    height: .482em;
+    width: .63em;
+  }
 
-    @include media-down(mobile) {
-      &.previous-page,
-      &.next-page {
-        float: none;
-        width: 100%;
-      }
+  .pagination-label {
+    display: inline-block;
+    margin-top: 0.1em;
+    text-decoration: underline;
 
-      &.next-page a {
-        text-align: right;
-      }
+    &:empty {
```

Comment:
I'm not sure, but in manuals it does render which suggests it's passing through something...